frontend/src/components/dropDown.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link } from 'react-router-dom';
+import { DownOutlined } from '@ant-design/icons';
+import type { MenuProps } from 'antd';
+import { Button, Dropdown, Space, Typography } from 'antd';
+
+interface DropDownProps {
+  username: string;
+  handleSignout: () => void;
+}
+function DropdownMenu({ username, handleSignout }: DropDownProps): JSX.Element {
+  const items: MenuProps['items'] = [
+    {
+      label: <Link to='/profile'>1st menu item</Link>,
```

Comment:
Improper link title, see attached screenshot.

![image](https://github.com/COSC481W-2023Fall/Bookworm/assets/43181046/66bbebd9-ccba-44ec-a323-b8d3427d25a9)

```suggestion
      label: <Link to='/profile'>Profile</Link>,
```